packages/rum-core/src/domain/view/viewMetrics/trackInteractionToNextPaint.ts

```diff
@@ -76,6 +84,20 @@ export function trackInteractionToNextPaint(
       } else {
         interactionToNextPaintTargetSelector = undefined
       }
+      if (
+        !interactionToNextPaintTargetSelector &&
+        isExperimentalFeatureEnabled(ExperimentalFeature.NULL_INP_TELEMETRY)
+      ) {
+        if (interactionSelectorMap.has(newInteraction.startTime)) {
+          interactionToNextPaintTargetSelector = interactionSelectorMap.get(newInteraction.startTime)
+          interactionSelectorMap.delete(newInteraction.startTime)
+        }
+      }
+      interactionSelectorMap.forEach((_, key) => {
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏ You could isolate this logic in a dedicated function like `deleteOldInteractionSelectorByTimestamp`

Reply:
I renamed it to `deleteOutdatedInteractionSelectors`. 'Interaction' feels a bit lengthy, maybe we can just call it `deleteOutdatedSelectors`. Wdyt?